kotlin/src/main/resources/detekt.yml

```diff
@@ -487,6 +489,8 @@ potential-bugs:
 
 style:
   active: true
+  ClassOrdering:
+    active: true
```

Comment:
Ça va tu obliger les fonctions à être en ordre alphabétique ?

Reply:
Non, mais juste l'ordre des éléments
https://detekt.github.io/detekt/style.html#classordering

1.     Property declarations and initializer blocks
2.     Secondary constructors
3.     Method declarations
4.     Companion object

Particulièrement content que companion object soit à la fin. Certains le mettent au début parce que c'est static en java, mais c'est noisy.

Reply:
Ah ouain, je met toujours mes companion object en haut justement parce que c'est comme du data statique